styleguide/components/31-molecules/islands/gentinfo-island/gentinfo-island.twig

```diff
@@ -11,11 +11,11 @@
     %}
     {% include '@list'
       with {
-        'type': 'unordered',
-        'items': [
-          '<a href="tel:09 210 10 10" alt="Gentinfo tel link" class="button button-secondary button-call icon-phone">T: 09 210 10 10</a>',
-          '<a href="mailto:gentinfo@stad.gent" alt="Gentinfo mail link">Gentinfo@stad.gent</a>',
-          '<a href="javascript:(void);" class="standalone-link" alt="Contact form">Contactformulier</a>'
+        'type': 'icon-list',
+        'icon_items': [
+          '<i class="icon-phone" aria-hidden="true"></i><a href="tel:09 210 10 10" alt="Gentinfo tel link">09 210 10 10</a>',
```

Comment:
trek het alvast door op de Drupal theming :) href hoort in internationaal format te zijn